### ISSUE TITLE
Tickertap doesn't show the right icon

### ISSUE THREAD

**Issue by HavannaFR** [matextrem/streamdeck-tickertap#26]
**Describe the bug**
When 2 cryptocurrencies have the same name, it is not necessarily the right icon that is displayed even if the values ​​remain correct.

**To Reproduce**
1. Add new token like "VELODROME-FINANCE" (https://coinmarketcap.com/currencies/velodrome-finance/)
2. The displayed icon will be from this token https://coinmarketcap.com/currencies/velo/

**Expected behavior**
Should be the icon from https://coinmarketcap.com/currencies/velodrome-finance/.

**Screenshots**
https://i.imgur.com/5w9SNGB.png

**Comment by matextrem** [comment]
Hey @HavannaFR! Thanks for reporting the issue. 

Unfortunately, we're using a third party repository for Stocks/Crypto images and the one you want is not included there https://github.com/nvstly/icons/tree/main/crypto_icons 

So I suggest you creating a PR in their repo to include this new crypto logo. It should also have a custom name other than VELO as it already exists and it's pointing to their logo. 

Regards, 

Mati

**Comment by HavannaFR** [comment]
Hi @matextrem, thanks for your quick reply !

I saw you are using ".flexStart.alignBaseline .base-text" for getting the price directly from coinmarketcap page, why not using the same process for getting the icon instead ? If you get "section-coin-overview" you have the icon and the price elements inside 

Have a great day,
Havanna

**Comment by matextrem** [comment]
Hey @HavannaFR! 

We've released `TickerTap v1.9.1` which is now fetching Crypto icons directly from `CoinMarketCap` site. You can download it from [here](https://github.com/matextrem/streamdeck-tickertap/releases/tag/v1.9.1). Let me know if it works for you. 

Regards,

Mati

**Comment by HavannaFR** [comment]
Hi @matextrem !

Thanks for this quick update ! But the plugin doesn't seem to be up to date on Elgato Marketplace yet

Havanna

**Comment by matextrem** [comment]
@HavannaFR No, I've not uploaded to the Elgato marketplace yet. Just wanted to get confirmation from you that it's working as expected.

**Comment by HavannaFR** [comment]
Oh I didn't even know that you could manually install a Streamdeck plugin, I just tested it and it seems to work perfectly, thanks again @matextrem ! Could be awesome to add custom url icon field in a future update!
https://i.imgur.com/qf2KMC6.png
